docs/cmd-docker-doctor.md

```diff
@@ -0,0 +1,21 @@
+# `docker:doctor` command
+
+Diagnose problems with Docker setup and attempt to fix them.
+
+## Usage
+
+```
+dock-cli docker:doctor --dry-run
+dock-cli docker:doctor
+```
+
+## Available options
+
+- `--dry-run` don't attempt to fix the problems, just show problems and suggest
+  appropriate solutions
+
+## What does it do?
+
+- Test whether docker is installed and running.
```

Comment:
Is that not too much unnecessary detail? Ideally, the user shouldn't care which concrete tests are being run...